R/rename_with_counts.R

```diff
@@ -1,16 +1,25 @@
 #' Concatenates the count to the end specific variable value
 #'
-#' @description It can be useful in plots and tables to display the fequency of
-#'   categories. `rename_with_counts` concatenates the count of each variable
-#'   to its variable value to make these plots and tables.
+#' @description `rename_with_counts` concatenates the count of each category to
+#'   its category name given a specific variable. It can be useful in plots and
+#'   tables to display the fequency of categories of a variable (see the example
+#'   below).
 #' @param d a tibble or dataframe
 #' @param variable_name the column with counts wanted
 #' @return a tibble with the counts appended to the `variable_name` column
 #' @export
 #'
 #' @examples
+#' # TODO::
+#' rename_with_counts(pima_diabetes, weight_class)
+#'
+#' # Below is an example of how `rename_with_counts` can be helpful when
+#' # creating plots and tables. This graph shows the outcomes of different
+#' # weightclasses in `pima_diabetes`. With the added information from
+#' # `rename_with_counts`, we more correctly draw conclusions about the
```

Comment:
I'm not sure what "we more correctly draw conclusions..." means. Maybe "we can see how common each category is"?